pkg/shared/serviceregistryutil/error.go

```diff
@@ -0,0 +1,15 @@
+package serviceregistryutil
+
+import (
+	"fmt"
+)
+
+func NotFoundByIDError(id string) error {
+	NotFoundByIDErr := fmt.Errorf(`Registry instance with ID "%v" not found`, id)
```

Comment:
It wasn't possible to add localization to those (same as kafka) due to them being used in different context that do not have localiser injected. I think it is good to have those 2 localized and will add that after this PR (it will require a looot of changes that will obfuscate this PR)

Reply:
You could create a ServiceRegistryErrors struct to wrap these errors?